56 set avatar

gorgonov opened the pull request:
https://education.planfix.ru/task/56

Commit:
1) Создана миграция в таблице user: city, avatar, удалено поле photo_path из teacher_profile

Commit:
1) Создана миграция в таблице user: city, avatar, удалено поле photo_path из teacher_profile

PolyanskiyArt commented on app/models/Payment.php:
```diff
@@ -3,6 +3,7 @@
 namespace app\models;
 
 use Yii;
+use yii\web\UploadedFile;
```
эта строка нужна?

gorgonov replied:
нет, удалю.

PolyanskiyArt commented on app/modules/admin/controllers/UsersController.php:
```diff
@@ -2,118 +2,144 @@
+         */
+        $model = $this->findModel($id);
+
+
```
лишняя строка

gorgonov replied:
нет, иначе не будет экземпляра объекта модели. дальше она уже используется.

PolyanskiyArt commented on app/modules/admin/controllers/UsersController.php:
```diff
@@ -2,118 +2,144 @@
+//                return $this->refresh();
+            }
+
+            return $this->redirect(['update', 'id' => $model->id]);
```
можно ли без редиректа? с помощью vue динамически подгрузить изображение

gorgonov replied:
так тут не только изображение, все поля мошут поменяться.

PolyanskiyArt reviewed (commented):
есть замечания/вопросы

Commit:
1) Создана миграция в таблице user: city, avatar, удалено поле photo_path из teacher_profile

Commit:
Перенес аватар в верхнюю часть формы

Commit:
Перенес аватар в верхнюю часть формы

Commit:
Выполнено
https://education.planfix.ru/task/56

Commit:
Выполнено
https://education.planfix.ru/task/67
Исправлены фикстуры и тесты.
Тесты выполняются без ошибок

PolyanskiyArt committed:
Merge branch 'master' into 56_set_avatar
